### ISSUE TITLE
Version:fatal

### ISSUE THREAD

**Issue by ronrosenfeld** [SickChill/sickchill.github.io#10]
Just noticed this today.  I had recently done a "clean" install of sickchill running under Docker, and set up a new DB by "adding shows from existing".

All seems to work well.

But I see:

Branch: master 
Commit: f7a113a737654a91076bb42221e119db6258a276 
Version: fatal: No names found, cannot describe anything. 
Database Version: 44.0 

Before today's update, the Version line still had `fatal`, but also had something about no tag associated with some long alphanumeric code.

**Comment by BenjV** [comment]
Most likely a permission problem.
What system are you using to run the docker on?

**Comment by ronrosenfeld** [comment]
I’m running docker on a Synology NAS

**Comment by BenjV** [comment]
Just install the package on your Nas  and forget about docker.
Docker is a useless function who create only overhead and problems.
A Synology package can be found here,
https://github.com/BenjV/SYNO-packages/blob/master/SickChill%20noarch%20V1.1.spk

Don't forget to read this and act accordingly:
https://github.com/SynoCommunity/spksrc/wiki/Permission-Management

**Comment by ronrosenfeld** [comment]
Yeah,

I had used SickRage (the original) on my first NAS.  But it was really entry level and much too slow.  When I upgraded to the current model, there was no package that would run on that DSM.

 

So I migrated to Docker with NZBGet and first SickRage (the original), and now SickChill.  Things run well with that combination,  for my purposes.  The only issue I’ve come across is that Version issue.

 

I suppose, one of these days, I could re-install directly on the NAS, but don’t really have the time to do it right now.

 

Ron
